themes/mlibrary_new/exhibit-builder/exhibits/summary.php

```diff
@@ -3,6 +3,7 @@
   * Copyright (c) 2016, Regents of the University of Michigan.
   * All rights reserved. See LICENSE.txt for details.
   */
+  set_theme_option('display_header','1');
```

Comment:
Same as the other one, these are likely to be problematic here.

Reply:
Same issue, in this page as well, we do not want the banner image to show up.